CMakeLists.txt

```diff
@@ -48,15 +48,20 @@ ENDIF()
 IF(${ENABLE_SCRIPT_DEBUG})
 	add_definitions(-DSCM_DEBUG_INSTRUCTIONS)
 ENDIF()
+find_package(OpenGL REQUIRED)
+find_package(Bullet REQUIRED)
+find_package(SFML 2 COMPONENTS system window audio graphics network REQUIRED)
+find_package(MAD REQUIRED)
+find_package(GLM REQUIRED)
+
+
 
 # Find dependancies
 IF(APPLE)
 	set(OPENRW_PLATFORM_LIBS iconv)
+	include_directories("${GLM_INCLUDE_DIRS}")
```

Comment:
This is technically required on all platforms, It would be best to move this line after the find_package directives and do it on all platforms.